### DISCUSSION TITLE
Machine learning

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#39] [Q&A] [answered]
What is supervised learning? Mention what type of dataset it will use?

**Comment by TRahulsingh** [comment]
Supervised learning is a type of machine learning where an algorithm learns from labeled data to make predictions. It uses datasets with input features (X) and corresponding output labels (y), like images labeled as "cat" or "dog", to learn patterns and make accurate predictions.

**Comment by 17-Manish** [comment, marked as answer]
Supervised learning is a type of ML where a model is trained using labeled data. This means each example in the dataset has an input (like a photo) and a known output or label (like “cat” or “dog”). The model learns by looking at these input-output pairs, so it can predict the correct output for new inputs.
types that are used
supervised learning 
unsupervised
semi-supervised learning
reinforcement learning
